### PR TITLE
Form Validation

### PR DESCRIPTION
**Pull request by felixodette**
Implement a simple validation:

- The content of the email field has to be in lower case.
- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,172 +68,186 @@ <h1>Felix Odete</h1>
...
+<!--                    </div>-->
+<!--                    <div class="javascript-rectangle">-->
+<!--                        <p>Javascript</p>-->
+<!--                    </div>-->
+<!--                    <div class="see-project">-->
+<!--                        <h3>See this project</h3>-->
+<!--                        <img alt="" src="img/union.svg">-->
+<!--                    </div>-->
+<!--                </div>-->
+<!--            </div>-->
```
Comment (aimemalaika) [inline index.html]: - [x] Kindly remove commented code, commented or unused code reduce readability and maintainability of your program.

File: js/app.js [hunk 2]
```diff
@@ -6,9 +6,240 @@ const navSlide = () => {
...
+  } else {
+    localStorage.clear();
+    const userInfo = {
+      name: document.querySelector('#list-name').value,
+      email: document.querySelector('#list-email').value,
+      messge: document.querySelector('#list-textarea').value,
+    };
+    localStorage.setItem('userdata', JSON.stringify(userInfo));
+  }
+});
```
Comment (aimemalaika) [inline js/app.js]: - [x] The contact form contains at least one validation checking if the email is written in all lowercase. Kindly work on that.

File: index.html [hunk 3]
```diff
@@ -246,15 +262,37 @@ <h2>Get started</h2>
...
+                <button data-see-live class="btn-see-live" type="button">See live <i class="fa fa-external-link"></i></button>
+                <button data-see-source class="btn-see-source" type="button">See source <i class="fa fa-github"></i></button>
+                <a href="" id="previous"><i class="fa-solid fa-arrow-left-long"></i>Previous project</a>
+                <a href="" id="next">Next project<i class="fa-solid fa-arrow-right-long"></i></a>
+            </div>
+        </div>
+        <div id="overlay"></div>
     </main>
     <footer>
         <h3>Get my Resume</h3>
```
Comment (aimemalaika) [inline index.html:182]: - [x] Kindly fix the white space under the footer and remove the commented code inside the footer. chek the illustration bellow 
Screen Shot 2022-04-15 at 11 46 24

File: index.html [hunk 4]
```diff
@@ -3,20 +3,22 @@
...
     <link href="css/style.css" rel="stylesheet">
+    <link rel="stylesheet" href="css/modal.css">
     <link href='https://fonts.googleapis.com/css?family=Playfair+Display' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://use.fontawesome.com/75d401af0c.js"></script>
     <title>Felix Odete's Portfolio</title>
     <meta content="width=device-width, initial-scale=1" name="viewport">
 </head>
 <body>
```
Comment (aimemalaika) [inline index.html:14]: - [x] Kindly fix the website desktop version responsively as some part of the page are overflowing
Screen Shot 2022-04-15 at 11 50 05

File: index.html [hunk 5]
```diff
@@ -246,15 +262,37 @@ <h2>Get started</h2>
...
+            </div>
+            <div class="modal-body">
+                <img src="#" alt="Project image" class="rectangle-34" id="image-1">
+                <img src="#" alt="Project image" class="rectangle-39" id="image-2">
+                <img src="#" alt="Project image" class="rectangle-40" id="image-3">
+                <img src="#" alt="Project image" class="rectangle-41" id="image-4">
+                <img src="#" alt="Project image" class="rectangle-42" id="image-5">
+                <div data-html-css class="html-css html-border"></div>
+                <div data-ruby-rails class="ruby-rails ruby-border"></div>
+                <div data-javascript class="javascript javascript-border"></div>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly the error message should show up close to the "submit" button. when a validation error occurs.

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @felixodette, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights


## Required Changes ♻️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (felixodette) [reply index.html]: Thanks! Removed the commented code!

File: index.html (hunk 3 above)
Comment (felixodette) [reply index.html:182]: Removed the whitespace and commented code. Thanks!

File: js/app.js (hunk 2 above)
Comment (felixodette) [reply js/app.js]: Added validation for lowercase email.

Screenshot 2022-04-15 at 14 38 13

File: index.html (hunk 4 above)
Comment (felixodette) [reply index.html:14]: Thanks for the observation. Looked into it and fixed it.

Screenshot 2022-04-15 at 14 40 02

File: js/app.js [hunk 6]
```diff
@@ -6,6 +6,182 @@ const navSlide = () => {
...
+    showError();
+  }
+});
+
+form.addEventListener('submit', (event) => {
+  if (!email.validity.valid) {
+    showError();
+    event.preventDefault();
+  }
+});
```
Comment (phelian23) [inline js/app.js]: - [x] Good job working on the validation. I can see you implement it here. However, when I continue typing the email, the error no longer shows, and the form submits anyway even when the email contains uppercase letters. Check the video I recorded. Kindly fix this.
https://www.loom.com/share/0aa539a20a004515b18aafa0f22a475f

- [x] Also the error message should be placed close to the submit button as requested by the previous reviewer. Kindly fix it.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @felixodette, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/app.js (hunk 6 above)
Comment (felixodette) [reply js/app.js]: Hey @phelian23 fixed both issues. Thanks for the quick review.

Screenshot 2022-04-15 at 18 30 23

**Review by AnselemOdims — APPROVED** [review]
### Hi  @felixodette 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**